templates/springnative/cloudevents/src/main/java/functions/SpringCloudEventsApplication.java

```diff
@@ -0,0 +1,81 @@
+package functions;
+
+import static org.springframework.cloud.function.cloudevent.CloudEventMessageUtils.ID;
+import static org.springframework.cloud.function.cloudevent.CloudEventMessageUtils.SOURCE;
+import static org.springframework.cloud.function.cloudevent.CloudEventMessageUtils.SPECVERSION;
+import static org.springframework.cloud.function.cloudevent.CloudEventMessageUtils.SUBJECT;
+
+import java.util.UUID;
+import java.util.function.Function;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.function.cloudevent.CloudEventHeaderEnricher;
+import org.springframework.cloud.function.web.util.HeaderUtils;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.HttpHeaders;
+import org.springframework.messaging.Message;
+
+@SpringBootApplication
+public class SpringCloudEventsApplication {
+
+  private static final Logger LOGGER = Logger.getLogger(
+      SpringCloudEventsApplication.class.getName());
+
+  public static void main(String[] args) {
+    SpringApplication.run(SpringCloudEventsApplication.class, args);
+  }
+
+  @Bean
+  public Function<Message<Input>, Output> uppercase(CloudEventHeaderEnricher enricher) {
```

Comment:
I recommend adding a lot of comments here, and in the function body indicating where devs should add their code, and what the parameters to the function are, etc. If you don't @omerbensaadon will mention it. :wink: